crates/daphne/src/pine/vdaf.rs

```diff
@@ -981,14 +988,14 @@ mod tests {
         };
 
         let got = PinePrepState::get_decoded_with_param(
-            &(&pine, true),
+            &(&pine, DapAggregatorRole::Leader),
```

Comment:
I'd prefer this type not bleed into the `pine` module. Ideally this module could be used externally without having to use the rest of the crate.